recipes/vibrant/download-db.sh

```diff
@@ -31,10 +31,10 @@ echo "Downloading VIBRANT databases to ${VIBRANT_DATA_PATH}..."
 
 # use Python script to download and test data
 cd ${VIBRANT_DATA_PATH}/databases
-python VIBRANT_setup.py
+python3 VIBRANT_setup.py
 
-echo "Testing VIBRANT databases..."
-python VIBRANT_test_setup.py
+#echo "Testing VIBRANT databases..."
```

Comment:
Just remove the lines instead of commenting. The only reason you might use a comment is if you think this change is only temporary in this version of the recipe. But then you need to explain under which circumstances these lines should be uncommented in the future or when they can be removed.